### ISSUE TITLE
Adjust daily recommended calories

### ISSUE THREAD

**Issue by grrlguide** [davidhealey/waistline#601]
There is correctly no way to modify the 2000 calorie per day recommended intake. Odd like to be able to modify this as well as adjust the recommended balances of proteins and carbs to follow my correct dietary guidelines for my weight, she and activity level.

**Comment by EmilJunker** [comment]
You can change these values in the goal editor. Just open the hamburger menu and tap on Goals.

In the next release, there will be a link under `Settings > Goals` that takes you directly to the goal editor. This should hopefully make that feature easier to discover :)

**Comment by mikjori** [comment]
Would it be possible to implement some kind of automatic calculation for this thing? Like those found on the internet, where you enter your age, height, weight etc. and then calculate the amount of calories needed normally or for weight loss.

**Comment by davidhealey** [comment]
Yes. Actually there is a TDEE calculator inside Waistline, but it's currently hidden because I haven't finished implementing it. When I have some free time to do more development work on Waistline it's at the top of my list.

**Comment by mikjori** [comment]
Happy to hear that! Take your time.
